src/components/NavigationDrawer.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <v-navigation-drawer app v-model="drawerState" style="max-height:none">
     <v-list>
-      <v-list-tile @click="routeTo('/')">
+      <v-list-tile @click="getRouteData()" to='/' active-class="primary--text nav-active">
```

Comment:
How does it remove the classes if we go to different pages? As I see it whenever we go to a category it will leave the last one selected as well. Same goes for refreshing the page, does it show the active class on the right category on refresh?